### PR TITLE
feat: Onboard JSS 22.1 (GenAI Knowledge Base)

### PR DESCRIPTION
**Pull request by NimJay**
* JSS = Jump Start Solutions
* JSS 22.1 = GenAI Knowledge Base
* For more info, see Google-internal [CL for the solution guide](http://cl/609804270).
* CC: @davidcavazos (Info about onboarding as a Blueprint: [go/jss-authoring-guide/develop-code-for-a-solution.md#onboard-as-a-blueprint](go/jss-authoring-guide/develop-code-for-a-solution.md#onboard-as-a-blueprint))

### CODE REVIEW

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -743,6 +743,15 @@ locals {
       groups          = ["dee-data-ai", local.jss_common_group]
       enable_periodic = true
     },
+    {
+      name            = "terraform-genai-knowledge-base"
```
Comment (bharathkkb) [inline infra/terraform/test-org/org/locals.tf:747]: @NimJay could you grant `GoogleCloudPlatform/blueprint-solutions` admin access

**Review by bharathkkb — COMMENTED** [review]
Thanks for the PR!

File: infra/terraform/test-org/org/locals.tf (hunk 1 above)
Comment (NimJay) [reply infra/terraform/test-org/org/locals.tf:747]: Admin access has been granted to [the `GoogleCloudPlatform/blueprint-solutions` team](https://github.com/orgs/GoogleCloudPlatform/teams/blueprint-solutions).
(Thanks, @davidcavazos! :))

Comment (apeabody) [reply infra/terraform/test-org/org/locals.tf:747]: Thanks!

File: infra/terraform/test-org/org/locals.tf [hunk 2]
```diff
@@ -743,6 +743,15 @@ locals {
...
+      short_name      = "genai-doc-knowledge-base"
```
Comment (apeabody) [inline infra/terraform/test-org/org/locals.tf:748]: @NimJay - Should this perhaps be
```suggestion
      short_name      = "genai-doc-knowledge-base"
```

Comment (apeabody) [inline infra/terraform/test-org/org/locals.tf:748]: This just needs to match the value in `infra/terraform/test-org/ci-triggers/locals.tf`
```suggestion
      short_name      = "genai-knowledge-base"
```

Comment (apeabody) [reply infra/terraform/test-org/org/locals.tf:748]: @NimJay

**Review by apeabody — APPROVED** [review]

**Pull request merged**